app/views/website_members/index.html.haml

```diff
@@ -18,9 +19,13 @@
             %th.text-right Actions
         %tbody.websites
           - @website_members.each do |website_member|
-            %tr
-              %td= website_member.user.name
-              %td= website_member.user.email
-              %td= website_member.role
-              %td.text-right
-                = link_to ' Delete', "/website_members/#{website_member.id}", :method => :delete, data: {confirm: "Are you sure?"}, :class => "delete_website btn btn-danger pull-right fa fa-close"
\ No newline at end of file
+            - unless @current_website_member == website_member
+              = simple_form_for website_member, wrapper: :vertical_form do |f|
+                %tr
+                  %input{:name => "id", :type => "hidden", :value => "#{website_member.id}"}/
+                  %td= website_member.user.name
+                  %td= website_member.user.email
+                  %td
+                    = f.input :role, label: false, collection: [['Owner', 1], ['User', 2]], selected: website_member.role.value, input_html: {:style => "max-width: 90px;"}
+                  %td.text-right
```

Comment:
Why do you have a form if you are not submitting anything. This is also wrong as it's in a table. There is no reason for that form there you are doing an ajax call...